.cirrus.yml

```diff
@@ -80,6 +80,9 @@ qa_task:
     <<: *CONTAINER_DEFINITION
     cpu: 4
     memory: 8G
+  node_version_script:
+    - node --version
+    - npm --version
```

Comment:
Was this added for debugging purposes or do you intentionally need these?

Reply:
it's for debugging, but I would keep them, it makes it easier to understand the version of node used

Reply:
👍